node_modules/.bin/js-yaml

```diff
@@ -1 +0,0 @@
-../js-yaml/bin/js-yaml.js
```

Comment:
here's an example of npm `"scripts"` to use `dotenv` for loading `.env` files, if that's still needed:

```json
        "run": "node -r dotenv/config src/app.js",
        "build": "ncc build ./index.js --license licenses.txt",
        "lint": "eslint src/",
        "test": "jest --passWithNoTests ."
```